File: erp-frontend/src/api/sc/stock/product-stock-batch/model/queryProductStockBatchVo.ts
import { PageVo } from '@/api/model/pageVo';

export interface QueryProductStockBatchVo extends PageVo {
  /**
   * 仓库ID
   */
  scId?: string;

  /**
   * 航材编号
   */
  productCode?: string;

  /**
   * 航材名称
   */
  productName?: string;

  /**
   * 航材分类ID
   */
  categoryId?: string;

  /**
   * 航材制造商ID
   */
  brandId?: string;

  /**
   * 批次号
   */
  batchNumber?: string;
}

File: erp-backend/xingyun-sc/src/main/java/com/lframework/xingyun/sc/vo/stock/batch/QueryProductStockBatchVo.java
package com.lframework.xingyun.sc.vo.stock.batch;

import com.lframework.starter.web.core.vo.PageVo;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

/**
 * 查询批次库存的参数
 */
@Data
public class QueryProductStockBatchVo extends PageVo {

    private static final long serialVersionUID = 1L;

    /**
     * 仓库ID
     */
    @ApiModelProperty(value = "仓库ID")
    private String scId;

    /**
     * 商品编号
     */
    @ApiModelProperty(value = "商品编号")
    private String productCode;

    /**
     * 商品名称
     */
    @ApiModelProperty(value = "商品名称")
    private String productName;

    /**
     * 商品类目ID
     */
    @ApiModelProperty(value = "商品类目ID")
    private String categoryId;

    /**
     * 商品品牌ID
     */
    @ApiModelProperty(value = "商品品牌ID")
    private String brandId;

    /**
     * 批次号
     */
    @ApiModelProperty(value = "批次号")
    private String batchNumber;
}
